torch/_dynamo/convert_frame.py

```diff
@@ -1092,6 +1100,7 @@ def format_guard_failures() -> str:
                 config.suppress_errors,
                 config.inline_inbuilt_nn_modules,
                 config.specialize_float,
+                json.dumps(config_dict),
```

Comment:
I don't see evidence that any filtering has happened here; how big is this string right now?

Reply:
Yes, updated to list.